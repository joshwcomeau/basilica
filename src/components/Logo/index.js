/* eslint-disable react/no-danger */
// eslint-disable-next-line no-unused-vars
import React, { Component, PropTypes } from 'react';
import classNames from 'classnames';

import logo from '../../images/logo.svg';
import './index.scss';


const Logo = () => {
  const classes = classNames(['logo']);

  return (
    <div className={classes} dangerouslySetInnerHTML={{ __html: logo }} />
  );
};

Logo.propTypes = {

};

Logo.defaultProps = {

};

export default Logo;
