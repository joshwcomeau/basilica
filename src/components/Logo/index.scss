@import '../../variables';

.logo {
  display: block;
  width: 145px;
  height: 60px;
  display: flex;
  align-items: center;

  svg {
    height: 40px;
  }

  @media screen and (max-width: $mq-tablet) {

  }

  @media screen and (max-width: $mq-mobile) {

  }
}
