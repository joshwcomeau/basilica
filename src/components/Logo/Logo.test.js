/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Logo from './index';

it('renders correctly', () => {
  const tree = renderer.create(<Logo />).toJSON();
  expect(tree).toMatchSnapshot();
});
