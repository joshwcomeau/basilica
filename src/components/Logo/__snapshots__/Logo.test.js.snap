exports[`Logo renders correctly 1`] = `
<h2
  className="logo">
  Basilica
</h2>
`;
